### PR TITLE
Add Hack Club Semester 2 Email Update

### LINKED ISSUE #458: Write a spring semester update to our clubs
_From @jonleung on January 6, 2016 20:20_

_Copied from original issue: hackclub/hackclub#640_

### PR DESCRIPTION
**Pull request by jonleung**
This closes #458

### CODE REVIEW

**Comment by zachlatta** [comment]
Here's a quick edit of this I wrote:

> Hi everyone!
> 
> We've learned and failed a tremendous amount this past semester. All of you
> have been incredibly supportive and both of us feel so privileged to have the
> opportunity to work with each and every one of you.
> 
> In the next two months, Hack Club is focusing on growth. Our goal is to start
> 50 clubs and increase our weekly attendance by 750 students by the end of
> February. As a result of this, we're focusing on rewriting our early workshops
> and standardizing the beginning of clubs ([click here][club_guide] for an
> example of what we're working on). This means that we won't be building
> curriculum for clubs like yours at the pace we did last semester and we may
> not have new workshops in time for all of your meetings while we're still
> early into this semester.
> 
> Your work this past semester has led us to where we are today and we'll only
> be able to continue the movement with your continued involvement. As we
> onboard new clubs, the best way for you to contribute is to welcome and
> support new club leaders. Make the Slack feel like it's their home. We're
> looking to pair existing clubs with new clubs to start the mentorship early.
> If you are interested in being a mentor to new clubs, please reply directly to
> this email to let us know.
>
> Let's make 2016 amazing!
> Jonathan, Zach
>
> [club_guide]: https://gh.hackclub.io/clubs/README.md

**Comment by jonleung** [comment]
@zachlatta, it looks good! Do you want to

1. submit this as a pull request
2. close my pull request in favor of your pull request
3. and go ahead and send this out?

**Comment by zachlatta** [comment]
Can you update the pull request with my edit and I'll go ahead and take point on sending this out?

--
Zach Latta
Hack Club | zach@hackclub.io

**Comment by jonleung** [comment]
I just updated the pull request with your edits!

File: labs/spring_2016_update.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Subject: You Are Now Hack Club's Ambassadors
```
Comment (zachlatta) [inline labs/spring_2016_update.md]: What if we changed the subject to "Hack Club Update" or "Hack Club Semester 2 Update"? I don't think we're turning our Labs club leaders into "ambassadors" with this email.

File: labs/spring_2016_update.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+In the next two months, Hack Club is focusing on growth. Our goal is to start
+50 clubs and increase our weekly attendance by 750 students by the end of
+February. As a result of this, we're focusing on rewriting our early workshops
+and standardizing the beginning of clubs ([click here][club_guide] for an
+example of what we're working on). This means that we won't be building
+curriculum for clubs like yours at the pace we did last semester and we may
+not have new workshops in time for all of your meetings while we're still
+early into this semester.
+
+[club_guide](https://github.com/hackclub/hackclub/blob/master/clubs/README.md)
```
Comment (zachlatta) [inline labs/spring_2016_update.md]: Reference link definitions should go at the bottom of the section (and in this case, the file). Can you move this to the very bottom?

Comment (zachlatta) [inline labs/spring_2016_update.md]: Also, this should be formatted like this:

```md
[club_guide]: https://github.com/hackclub/hackclub/blob/master/clubs/README.md
```

**Comment by zachlatta** [comment]
Just left a couple more comments.

On Mon, Jan 11, 2016 at 10:28 AM, Jonathan Leung <notifications@github.com>
wrote:

> I just updated the pull request with your edits!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/meta/pull/459#issuecomment-170643123>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by zachlatta** [comment]
Can you also update the pull request's title to reflect that you're adding the entire update, and not just the outline?

File: labs/spring_2016_update.md (hunk 1 above)
Comment (jonleung) [inline labs/spring_2016_update.md]: I opted to change it to:

> Hack Club Semester 2 Update

File: labs/spring_2016_update.md (hunk 2 above)
Comment (jonleung) [inline labs/spring_2016_update.md]: I will make your suggested changes.

**Comment by jonleung** [comment]
@zachlatta, I have updated the title of the pull request!

File: labs/spring_2016_update.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+In the next two months, Hack Club is focusing on growth. Our goal is to start
+50 clubs and increase our weekly attendance by 750 students by the end of
+February. As a result of this, we're focusing on rewriting our early workshops
+and standardizing the beginning of clubs ([click here][club_guide] for an
+example of what we're working on). This means that we won't be building
+curriculum for clubs like yours at the pace we did last semester and we may
+not have new workshops in time for all of your meetings while we're still
+early into this semester.
+
+
```
Comment (zachlatta) [inline labs/spring_2016_update.md]: Is this extra newline intentional?

File: labs/spring_2016_update.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+onboard new clubs, the best way for you to contribute is to welcome and
+support new club leaders. Make the Slack feel like it's their home. We're
+looking to pair existing clubs with new clubs to start the mentorship early.
+If you are interested in being a mentor to new clubs, please reply directly to
+this email to let us know.
+
+Let's make 2016 amazing!
+Jonathan & Zach
+
+[club_guide] https://github.com/hackclub/hackclub/blob/master/clubs/README.md
```
Comment (zachlatta) [inline labs/spring_2016_update.md]: The reference link definition is missing a colon. Should be `[club_guide]: ...`

File: labs/spring_2016_update.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Your work this past semester has led us to where we are today and we'll only
+be able to continue the movement with your continued involvement. As we
+onboard new clubs, the best way for you to contribute is to welcome and
+support new club leaders. Make the Slack feel like it's their home. We're
+looking to pair existing clubs with new clubs to start the mentorship early.
+If you are interested in being a mentor to new clubs, please reply directly to
+this email to let us know.
+
+Let's make 2016 amazing!
```
Comment (zachlatta) [inline labs/spring_2016_update.md]: Should the end of this line have two spaces at the end to force the Markdown rendered to add a newline? See the "break" part of https://daringfireball.net/projects/markdown/syntax#p.

**Comment by jonleung** [comment]
@zachlatta, I have made your suggested changes! Assigning back to you!

**Comment by jonleung** [comment]
Oops, I accidently closed this. Reopening.

**Comment by jonleung** [comment]
@zachlatta, I have made the fixes that you suggested and have assigned this back to you.

**Comment by zachlatta** [comment]
I don't see them?

File: labs/spring_2016_update.md (hunk 3 above)
Comment (jonleung) [inline labs/spring_2016_update.md]: No! Will fix!

File: labs/spring_2016_update.md (hunk 5 above)
Comment (jonleung) [inline labs/spring_2016_update.md]: Oh that's cool, I didn't know about that!

Comment (jonleung) [inline labs/spring_2016_update.md]: Fixed

File: labs/spring_2016_update.md (hunk 4 above)
Comment (jonleung) [inline labs/spring_2016_update.md]: Fixed!

**Comment by jonleung** [comment]
@zachlatta, oops, i missed some of your comments earlier. I just fixed them and pushed my changes! Let me know if anything else needs fixing but I think we should bias for sending this sooner rather than later.

**Comment by zachlatta** [comment]
Sent!

**Comment by jonleung** [comment]
<3

**Pull request merged**